packages/sdk/src/schema/pages.ts

```diff
@@ -102,11 +102,31 @@ const ProjectMeta = z.object({
   code: z.string().optional(),
 });
 
+export const ProjectNewRedirectPathSchema = PagePath.or(
+  z.string().refine((data) => {
+    try {
+      return Boolean(new URL(data));
+    } catch {
+      return false;
+    }
+  }, "Must be a valid URL")
+);
+
+export const PageRedirectSchema = z.object({
+  old: PagePath,
+  new: ProjectNewRedirectPathSchema,
+});
+export const ProjectRedirectsSchema = z.array(PageRedirectSchema);
```

Comment:
Let's inline this type. Build array is not big deal but easy to confuse these types.